frontend/src/scenes/sessions/sessionsTableLogic.ts

```diff
@@ -10,6 +10,12 @@ import { sessionsFiltersLogic } from 'scenes/sessions/filters/sessionsFiltersLog
 
 type SessionRecordingId = string
 
+enum ExpandState {
+    expanded,
+    collapsed,
+    manual,
```

Comment:
This name is confusing - what does manual mean? When can something manually expand?

Reply:
Good point- what about [Uncontrolled](https://reactjs.org/docs/uncontrolled-components.html) or Unset?

Reply:
Both seem fine :+1: